form/2d/p5js/closethetriangle/index.js

```diff
@@ -0,0 +1,21 @@
+function setup() {
+  createCanvas (500,500);
+  colorMode (HSB, 360,100,100, 100);
+  background (150,75,90,100);
+  
+  
+  
+  fill (20);
+  rect (0,0,width/2,height);
+  
+  noStroke ();
+  fill (150,75,90,100);
+  triangle (150,100,100,300,380,250);
```

Comment:
actually it makes sense (or I would not ask for it). If somebody changes the size of the sketch your picture will not look the same